content/en/docs/internals/_index.md

```diff
@@ -166,8 +166,6 @@ represents the "internal" promotion to the stable stream prior to the execution
 of tests.  Dashed lines represent edges not fully depicted since they are
 optional and can be added to any image in the pipeline:
 
-- Each entry in `operator.substitutions` makes `src-bundle` depend on that
-  image.
```

Comment:
Agree that the next line expresses the same thing.
Could we remove the next one and keep the first line instead?
The first one is a better English expression to me. :)